examples/angular_bazel_architect/projects/frontend-lib/ng-package.js

```diff
@@ -4,7 +4,5 @@ const [outputPath] = process.argv.slice(-1);
 
 module.exports = {
   dest: resolve(outputPath),
-  lib: {
-    entryFile: 'src/public-api.ts'
-  }
+  lib: {entryFile: 'src/public-api.ts'}
```

Comment:
Change came from a "yarn format" which was requested by a pre-commit hook when I merged in the latest stable branch. Can revert if needed